lib/OpenQA/WebAPI/Controller/API/V1.pm

```diff
@@ -66,11 +66,12 @@ sub auth {
         }
     }
 
-    if ($user) {
+    if ($user && ($self->is_operator($user) || $self->is_admin($user))) {
         $self->app->log->debug(sprintf "API auth by user: %s, operator: %d", $user->username, $user->is_operator);
         $self->stash(current_user => {user => $user});
-        return $self->is_operator($user);
+        return 1;
```

Comment:
Not exactly that I make it available, it was available and I didn't forbid it (yet). Only thing this does is that regular user will see 403 instead of connection closed by timeout.